workers/unity/Packages/com.improbable.gdk.tools/LocalLaunch.cs

```diff
@@ -42,17 +54,156 @@ public static void BuildConfig()
             }
         }
 
-        public static void Launch()
+        public static void LaunchClient()
+        {
+            var command = Common.SpatialBinary;
+            var commandArgs = "local worker launch UnityClient default";
+
+            if (Application.platform == RuntimePlatform.OSXEditor)
+            {
+                command = "osascript";
+                commandArgs = $@"-e 'tell application ""Terminal""
+                                     activate
+                                     do script ""cd {SpatialProjectRootDir} && {Common.SpatialBinary} {command}""
+                                     end tell'";
+            }
+
+            var processInfo = new ProcessStartInfo(command, commandArgs)
+            {
+                CreateNoWindow = false,
+                UseShellExecute = true,
+                WorkingDirectory = SpatialProjectRootDir
+            };
+
+            var process = Process.Start(processInfo);
+
+            if (process == null)
+            {
+                Debug.LogError("Failed to start a standalone client locally.");
+                return;
+            }
+
+            process.EnableRaisingEvents = true;
+            process.Exited += (sender, args) =>
+            {
+                // N.B. This callback is run on a different thread.
+                if (process.ExitCode == 0)
+                {
+                    return;
+                }
+
+                var logPath = Path.Combine(SpatialProjectRootDir, "logs");
+                var latestLogFile = Directory.GetFiles(logPath, GetClientLogFilename())
+                    .Select(f => new FileInfo(f))
+                    .OrderBy(f => f.LastWriteTimeUtc).LastOrDefault();
+
+                if (latestLogFile == null)
+                {
+                    Debug.LogError($"Could not find a standalone client log file in {logPath}.");
+                    return;
+                }
+
+                var message = $"Unity Standalone Client local launch logfile: {latestLogFile.FullName}";
+
+                if (WasProcessKilled(process))
+                {
+                    Debug.Log(message);
+                }
+                else
+                {
+                    var content = File.ReadAllText(latestLogFile.FullName);
+                    Debug.LogError($"{message}\n{content}");
+                }
+
+                process.Dispose();
+                process = null;
+            };
+        }
+
+        private static string GetClientLogFilename()
+        {
+
+            var logConfigPath = Path.Combine(SpatialProjectRootDir, "workers", "unity");
+            var configFileJson = File.ReadAllText(Path.Combine(logConfigPath, ClientConfigFilename));
+            var dict = Json.Deserialize(configFileJson);
+            var defaultLogFileName = "unitygamelogic.log";
```

Comment:
this is the default name for the game logic worker. We want the default name for the client log file :)